demo/app/lib/widgets/credential_verified_information_view.dart

```diff
@@ -17,20 +16,16 @@ class CredentialVerifiedInformation extends StatelessWidget {
 
   Future<Widget> getCredentialDetails() async {
     List<CredentialPreviewData> list;
-    var data = json.decode(credentialData.credentialDisplayData!);
-    bool? hasOrder;
-    hasOrder = await WalletSDKPlugin.resolveOrder(credentialData.credentialDisplayData);
-    log("hasOrder $hasOrder");
-    var credentialClaimsData = data['credential_displays'][0]['claims'] as List;
+    var credentialDisplayData  = await WalletSDKPlugin.resolveCredDisplayRendering(credentialData.credentialDisplayData);
+    var credentialDisplayEncodeData = json.encode(credentialDisplayData);
+    List<dynamic> responseJson = json.decode(credentialDisplayEncodeData);
+    var credentialClaimsData = responseJson.first['claims'] as List;
     list = credentialClaimsData.map<CredentialPreviewData>((json) => CredentialPreviewData.fromJson(json)).toList();
-    if (hasOrder == true){
-      log("hasOrder $hasOrder");
-      list.sort((a, b) {
+    // TODO order function in the sdk is not working properly
```

Comment:
Can you provide more info? Are the values not returned from the function?

Reply:
@rolsonquadras  - `(BOOL)order:(long* _Nullable)ret0_ error:(NSError* _Nullable* _Nullable)error;` This is the function defination. As per golang implementation it should return int or long but rather i keep getting error () is not type of int. When i add this in the swift app crashes complaining there is no () type. Its empty parathenthis which i have not seen before.

Reply:
^ @DRK3

Reply:
https://github.com/trustbloc/wallet-sdk/issues/326 issue is created

Reply:
Have to pass in a `UnsafeMutablePointer<Int>` as the first argument. It is weird, but it's likely for some sort of compatibility reason during the Go<->Obj C translation.